backend/opencv_backend.py

```diff
@@ -0,0 +1,49 @@
+import cv2
+
+from backend import Backend, PublishFunc
+from overlay import Canvas
+
+class OpenCVBackend(Backend):
+    """ Gets and displays video using the OpenCV (`cv2`) library.
+
+        This is intended for use with laptops with webcams, not on the
+        Raspberry Pi. """
+
+    def __init__(self, width: int, height: int, publish_recording_status_func: PublishFunc):
+        super().__init__(width, height, publish_recording_status_func)
+
+        self.webcam = None
+
+        # Time between video frames when running on OpenCV, in milliseconds
+        self.frametime = 17
+
+        self.base_canvas = Canvas(self.width, self.height)
+        self.data_canvas = Canvas(self.width, self.height)
+        self.message_canvas = Canvas(self.width, self.height)
+
+    def start_video(self) -> None:
+        self.webcam = cv2.VideoCapture(0)
```

Comment:
0 is a magic number. Create a variable called `camera_id` to make this more obvious. Would there be instances where the id might not be 0? See https://docs.opencv.org/3.4/d8/dfe/classcv_1_1VideoCapture.html#ad890d4783ff81f53036380bd89dd31aa

Reply:
0 means to use the default camera, as far as I can tell. I don't think we need to worry about the case where a computer has more than one webcam for the time being, so the default camera should be sufficient.

Reply:
Aight, sounds good to me!